shacl/dataset.jsonld

```diff
@@ -70,13 +79,9 @@
           "sh:path": {
             "@id": "schema:potentialAction"
           },
-          "sh:minCount": 1,
-          "sh:class": {
-            "@id": "schema:SearchAction"
-          },
-          "sh:node": {
-            "@id": "schema:SearchActionShape"
-          }
+          "sh:minCount": 2,
+          "sh:maxCount": 2,
+          "sh:node": "schema:PotentialActionShape"
```

Comment:
@ddeboer Can you take a look at this shape? Writing SHACL using JSON-LD is rather confusing to me 😅

What I want to express here is that a `potentialAction` property must have one `SearchAction` and one `FindAction`, each with its own `query`.

Reply:
I’d suggest to split `schema:PotentialActionShape` into two separate shapes: `schema:FindActionShape` and `schema:SearchActionShape`, each with a required `schema:query` property. While this doesn’t feel completely DRY, we can enforce that exactly one `SearchAction` and exactly one `FindAction` is available, which is what we want.

Reply:
Pushed an update to the SHACL file that _should_ take care or enforcing the correct data.

Reply:
Excellent! I've tested it and it seems to work :)